isce2_topsapp/solid_earth_tides.py

```diff
@@ -66,18 +66,17 @@ def get_gunw_attrs(gunw_path: str) -> Tuple[float, np.array, np.array,
     group = 'science/grids/imagingGeometry'
     with xr.open_dataset(gunw_path, group=group, engine='rasterio') as ds:
         z_meta = ds.heightsMeta.data
-        lat_meta = ds.y.data
-        lon_meta = ds.x.data
+        gt = ds.rio.transform()
         # convert angles to rad
         inc_angle = np.deg2rad(ds.incidenceAngle.data)
         az_angle = np.deg2rad(ds.azimuthAngle.data-90)
         solidtide_atr = {
-            'LENGTH': len(lat_meta),
-            'WIDTH': len(lon_meta),
-            'X_FIRST': lon_meta[0],
-            'Y_FIRST': lat_meta[0],
-            'X_STEP':  lon_meta[1] - lon_meta[0],
-            'Y_STEP': lat_meta[1] - lat_meta[0],
+            'LENGTH': az_angle.shape[1],
+            'WIDTH': az_angle.shape[2],
+            'X_FIRST': gt.c,
```

Comment:
Thanks for doing this - I think it will be easier for someone to look up conventions in [`affine`](https://github.com/rasterio/affine), then how the string is designed.